terminus-ui/link/src/link.component.spec.ts

```diff
@@ -1,23 +1,98 @@
+import { APP_BASE_HREF } from '@angular/common';
+import { Component, ViewChild } from '@angular/core';
+import { ComponentFixture, tick } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { RouterModule } from '@angular/router';
+
+import { createComponent } from '@terminus/ngx-tools/testing';
+
 import { TsLinkComponent } from './link.component';
+import { TsLinkModule } from './link.module';
+
+@Component({
+  template: `
+  <ts-link
+    [destination]="destination"
+    [isExternal]="isExternal"
+    [tabIndex]="tabIndex"
+  >
+  My Link Text
+  </ts-link>
+  `,
+})
+class TestHostComponent {
+  destination!: any;
+  isExternal!: boolean;
+  tabIndex!: number | undefined;
+
+  @ViewChild(TsLinkComponent)
+  linkComponent!: TsLinkComponent;
+}
 
 
 describe(`TsLinkComponent`, function() {
-  let component: TsLinkComponent;
+  let component: TestHostComponent;
+  let fixture: ComponentFixture<TestHostComponent>;
+  let link: HTMLElement;
+  let linkComponent: TsLinkComponent;
 
   beforeEach(() => {
-    component = new TsLinkComponent();
+    fixture = createComponent(TestHostComponent, [{provide: APP_BASE_HREF, useValue: '/my/app'}], [TsLinkModule, RouterModule.forRoot([])]);
+    component = fixture.componentInstance;
+    linkComponent = component.linkComponent;
+    fixture.detectChanges();
   });
 
 
-  it(`should exist`, () => {
-    expect(component).toBeTruthy();
+  test(`should exist`, () => {
+    expect(linkComponent).toBeTruthy();
+  });
+
+  describe(`isInternal`, () => {
+    test(`should default and retrieve`, () => {
+      link = fixture.debugElement.query(By.css('.c-link')).nativeElement as HTMLElement;
+      fixture.detectChanges();
+      component.isExternal = false;
+      component.destination = ['/#'];
+
+      expect(component.isExternal).toBeFalsy();
+      expect(link.classList).toContain('qa-link-internal');
+      expect(component.destination).toEqual(['/#']);
```

Comment:
Except `expect(link.classList)`, the other two expect clause are not useful since you're setting them a couple of lines above?